Products/SQLAlchemyDA/da.py

```diff
@@ -300,7 +300,8 @@ def getInfo(self):
         if wrapper is not None:
             d = self.sa_zope_wrapper().__dict__.copy()
             d['DSN'] = self.sa_zope_wrapper().dsn
-            for k in d.keys()[:]:
+            dd = d.copy()
+            for k in dd.keys():
```

Comment:
Why is this needed?

Reply:
the old one is cloning the list of keys. On python3.8 (at least) keys is of type dict_keys(['a']), and 'dict_keys' object is not subscriptable. That's because keys on dict couldn't be mutated while iterating..